source/common/mongo/codec_impl.cc

```diff
@@ -44,9 +43,10 @@ bool GetMoreMessageImpl::operator==(const GetMoreMessage& rhs) const {
 }
 
 std::string GetMoreMessageImpl::toString(bool) const {
-  return fmt::format("[GET_MORE id={} response_to={} collection='{}' return={} cursor={}]",
-                     request_id_, response_to_, full_collection_name_, number_to_return_,
-                     cursor_id_);
+  return fmt::format(
+      R"EOF({{"opcode": "OP_GET_MORE", "id": {}, "response_to": {}, "collection": "{}", "return": {}, )EOF"
```

Comment:
fmt uses `{}` to find the field to replace. `{{` is effectively escaping `{` so fmt doesn't start trying to parse